### PR TITLE
[SDESK-1562] Remove 'ends never' option from recurring events. Allow …

### PR DESCRIPTION
**Pull request by MarkLark86**
…only a system configured maximum number of items in a series.

### CODE REVIEW

File: server/features/steps/steps.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def steip_impl_store_indexed_item_to_ctx(context, tag, index):
 def step_impl_then_get_event_file(context):
     assert_200(context.response)
     data = get_json_data(context.response)
-    url = '/upload/%s/raw' % data['filemeta']['media_id']
+    url = '/upload-raw/%s' % data['filemeta']['media_id']
```
Comment (vied12) [inline server/features/steps/steps.py]: the endpoint has changed ? did someone already changed it in the code base ?

Comment (MarkLark86) [inline server/features/steps/steps.py]: This was changed in superdesk-core on Apr 14 with this commit: superdesk/superdesk-core@6e0fc66

Comment (vied12) [inline server/features/steps/steps.py]: so you should maybe change it in the planning code base too

Comment (MarkLark86) [inline server/features/steps/steps.py]: The planning codebase uses the angular `upload` service from superdesk-client-core, therefor there is no need to change anything in the planning codebase.

**Pull request merged**